instrumentation/github.com/gin-gonic/gin/otelgin/gin.go

```diff
@@ -122,6 +122,7 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 
 		// Record the server-side attributes.
 		var additionalAttributes []attribute.KeyValue
+		additionalAttributes = append(additionalAttributes, sc.Route(c.FullPath()))
```

Comment:
`c.FullPath()` needs to be checked that it is not empty.

https://pkg.go.dev/github.com/gin-gonic/gin#Context.FullPath

Reply:
Tests need to be added to ensure only non-empty strings are added.

Reply:
I thought it's ok for attribute value to be empty. see : https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/github.com/gin-gonic/gin/otelgin/gin.go#L94

Reply:
I checked prometheus [doc](https://prometheus.io/docs/concepts/data_model/): `Labels with an empty label value are considered equivalent to labels that do not exist.` 
Removing empty value label in the sdk side makes sense since it's useless. I'll update per your suggestion.